mavlink/qgroundcontrol
# Add NTRIP support to feed RCTM data

Adds support for NTRIP source for RTCM data.

Description
-----------
Work based on code of @TobinHall  and @WTPENGUIN 

Test Steps
-----------
Run the app. Enter NTRIP server details in settings. Connect to the vehicle, observe GPS received reporting status 6 - RTK Fixed.



By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
